src/index.js

```diff
@@ -40,7 +40,7 @@ export default function urlJoin(...parts) {
 
     // Join parts
     const partsStr = parts
-    .filter((part) => typeof part === 'string')
```

Comment:
Perhaps we should support more types. The filter here was just to remove `null/undefined` values. Having that said we can switch the filter to `part != null`.

Reply:
The only downside is that `false` will be part of the URL.

Reply:
If you pass an object or array the url will be invalid, right? I think it's best to keep it at string or number for the moment. What do you think?

Reply:
Agree